pages/auth0.py

```diff
@@ -25,6 +30,12 @@ def request_login_link(self, username):
         self.wait.until(expected.visibility_of_element_located(
             self._send_email_locator)).click()
 
+    def login_with_github(self, username, password):
+        self.find_element(*self._login_with_github_button_locator).click()
```

Comment:
It looks to me like we're switching to a page served by github.com after this click. If so, I have some concerns over how reliable our tests will be. We're at the mercy of github changing the HTML served, which could break our locators. I would at least suggest marking related tests as depending on a third party (so we could run them in isolation), and we should have a separate page object for the GitHub content.

Reply:
I created a new page object for github. I'm not really concerned about Github changing their HTML, as we can easily catch that in the daily test run and it should be a low effort to update the locators.